### DISCUSSION TITLE
How to map the + , - , * and / key to the keyboard (Macos) ?

### DISCUSSION THREAD

**Discussion by jongena** [hneemann/Digital#1103] [Q&A]
I was able to map the numbers to the push button keys, but so far I could not map the operations +,-, * and / to a qwerty keyboard
where these operations are on "shifted" positions ?  My device is a Mac running Monterey as OS.

**Comment by hneemann** [comment]
On my machine this works:

![z](https://user-images.githubusercontent.com/7190563/214397157-e313b815-08a0-49d7-91b4-d87e11ddfade.png)

**Reply by xotmatrix** [reply]
Less than ideal on Windows with a US keyboard.

![javaw_2023-01-24_20-42-06](https://user-images.githubusercontent.com/14209329/214461550-d58d9c88-45bb-40fd-95fa-db663d72b299.png)

EDIT:

OK, your example works with the Numeric Keypad (which I don't have on my normal keyboard).
